docs/edgex-taf.md

```diff
@@ -133,29 +131,46 @@ To write the automation testing for virtual device service, complete the followi
     In this document, we deploy all services using docker, so we must add the docker images to the docker-compose file, as illustrated below:
 
     ``` yaml
-    # TAF/utils/scripts/docker/device-service.yaml
+    # TAF/utils/scripts/docker/docker-compose.yml, which generated by get-compose-file.sh
     
-      device-virtual:
-        image: edgexfoundry/docker-device-virtual-go:master
-        ports:
-        - "59900:59900"
-        container_name: edgex-device-virtual
-        hostname: edgex-device-virtual
-        networks:
-          - edgex-network
-        environment:
-          REGISTRY_HOST: edgex-core-consul
-          CLIENTS_CORE_DATA_HOST: edgex-core-data
-          CLIENTS_CORE_METADATA_HOST: edgex-core-metadata
-          Service_Host: edgex-device-virtual
-        entrypoint: ["/device-virtual"]
-        command: ["--registry","--configDir=${CONFIG_DIR}"]
-        volumes:
-          - ${WORK_DIR}/TAF/config/${PROFILE}:${CONFIG_DIR}:z
-        depends_on:
-          - consul
-          - data
-          - command
+    device-virtual:
+      command:
+        - -cp=consul.http://edgex-core-consul:8500
+        - --registry
+        - --configDir=${CONFIG_DIR}
+      container_name: edgex-device-virtual
+      depends_on:
+        consul:
+          condition: service_started
+        data:
+          condition: service_started
+        metadata:
+          condition: service_started
+      environment:
+        EDGEX_SECURITY_SECRET_STORE: "false"
+        SERVICE_HOST: edgex-device-virtual
+      hostname: edgex-device-virtual
+      image: nexus3.edgexfoundry.org:10004/device-virtual:latest
+      networks:
+        edgex-network: null
+      ports:
+      - mode: ingress
+        host_ip: 0.0.0.0
+        target: 59900
+        published: "59900"
+        protocol: tcp
+      read_only: true
+      restart: always
+      security_opt:
+      - no-new-privileges:true
+      user: 2002:2001
+      volumes:
+      - type: bind
+        source: /${WORK_DIR}/TAF/config/device-virtual
+        target: ${CONFIG_DIR}
+        bind:
+          selinux: z
+          create_host_path: true
```

Comment:
This section might be changed constantly in edgex-compose.
Remove this example, and ask the user to retrieve the device-virtual section from 
https://github.com/edgexfoundry/edgex-compose/blob/<release version>/docker-compose.yml
for example, https://github.com/edgexfoundry/edgex-compose/blob/main/docker-compose.yml

Reply:
Update the example to only keep `volumes` and `command`.